### ISSUE TITLE
Добавить в ридми ссылки на плейлист / канал

### ISSUE THREAD

**Issue by orby-tech** [roll-over/unknown-interview#197]

**Comment by kot1kuz** [comment]
Правильно ли я понял что ты просто в реад ми хочешь добавить что то подобное 
"
##### Ссылка на канал @orby_tech

https://www.youtube.com/@orby_tech/videos
"

**Comment by kot1kuz** [comment]
##### Ссылка на плейлист рабочих созвонов 

[https://www.youtube.com/@orby_tech/videos](https://www.youtube.com/playlist?list=PLQfCv6Vomjn6aNXg4WuyBxg5C79t9hhQN)

**Comment by kot1kuz** [comment]
Получается так ?

#####  Ссылка на плейлист : рабочие созвоны 

[[https://www.youtube.com/@orby_tech/videos](https://www.youtube.com/playlist?list=PLQfCv6Vomjn6aNXg4WuyBxg5C79t9hhQN)
](https://www.youtube.com/playlist?list=PLQfCv6Vomjn6aNXg4WuyBxg5C79t9hhQN)

**Comment by kot1kuz** [comment]
предложил изменения ( вроде сам не могу )

**Comment by orby-tech** [comment]
Так-с, если речь про маркдаун, то там в квадратных скобочках название, а в круглых ссылка должна быть

И сейчас проверю доступы, вроде должны быть
